docs/en/engines/table-engines/integrations/kafka.md

```diff
@@ -65,6 +69,10 @@ Required parameters:
 
 Optional parameters:
 
+- `kafka_security_protocol` - Protocol used to communicate with brokers. Possible values: `plaintext`, `ssl`, `sasl_plaintext`, `sasl_ssl`.
+- `kafka_sasl_mechanism` - SASL mechanism to use for authentication. Possible values: `GSSAPI`, `PLAIN`, `SCRAM-SHA-256`, `SCRAM-SHA-512`, `OAUTHBEARER`.
+- `kafka_sasl_username` - SASL username for use with the `PLAIN` and  `SASL-SCRAM-..` mechanisms.
```

Comment:
Sorry for this, I know it is kind of nitpicking, but let's keep our docs tidy.

```suggestion
- `kafka_sasl_username` - SASL username for use with the `PLAIN` and `SASL-SCRAM-..` mechanisms.
```

Reply:
done